src/core/configuration/configurationService.ts

```diff
@@ -38,6 +39,20 @@ class ConfigurationService {
     return this.configurationEnv;
   }
 
+  private setKeriaIp() {
+    const keriaUrl = ConfigurationService.configurationEnv.keri.keria.url;
+    const keriaBootUrl =
+      ConfigurationService.configurationEnv.keri.keria.bootUrl;
+    if (keriaIP) {
```

Comment:
Is it possible in the webpack file to check for this variable too and put a very clear log in the terminal to show it's changed so people know in case they forgot to comment the env variable out when building?

Reply:
May I put a warning in the terminal in case they forgot to comment the env variable out when running dev?

Reply:
{
         apply: (compiler) => {
            compiler.hooks.compile.tap('[Warning] ', () => {
               if (process.env.KERIA_IP) {
                  console.warn('\n ⚠️ You are running the development server with the KERIA_IP environment variable set ⚠️\n\n');
               }
            });
         }
      },

Reply:
yep this would be perfect - also if we can print the value of `KERIA_IP` too